backend/tests/apps/mentorship/model/test_mentee_program.py

```
@@ -0,0 +1,85 @@
+            __str__=lambda self: "testmentee",
+        )
+        self.program = MagicMock(
+            spec=Program,
+            name="Test Program",
+            key="test-program",
+            __str__=lambda self: "Test Program",
+        )
+
+    @patch("apps.mentorship.models.MenteeProgram.objects.create")
+    def test_create_mentee_program(self, mock_create):
+        start_time = datetime(2023, 1, 1, 10, 0, 0)
```

Ruff (0.14.10)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)